packages/back-end/src/events/handlers/slack/slack-event-handler-utils.ts

```diff
@@ -371,6 +377,56 @@ const buildSlackMessageForExperimentDeletedEvent = (
   };
 };
 
+const buildSlackMessageForExperimentInfoSignificanceEvent = ({
+  metricName,
+  experimentName,
+  experimentId,
+  variationName,
+  statsEngine,
+  criticalValue,
+  winning,
+}: ExperimentInfoSignificancePayload): SlackMessage => {
+  const percentFormatter = (v: number) => formatNumber("#0.%", v * 100);
+
+  const text = ({
+    metricName,
+    variationName,
+    experimentName,
+  }: {
+    metricName: string;
+    variationName: string;
+    experimentName: string;
+  }) => {
+    if (statsEngine === "frequentist") {
+      return `In experiment ${experimentName}: metric ${metricName} for variation ${variationName} is
```

Comment:
Make experimentName here and in the "bayesian" case below a link to the experiment with the #results anchor.